src/components/SetupPanes/ForLiquidityPool.js

```diff
@@ -0,0 +1,20 @@
+import React from "react";
```

Comment:
the next 3 files are `.js`. Since they're new, do you think we should implement them in TS?